meta-avocado/recipes-avocado/packagegroups/packagegroup-avocado-sdk-extra.bb

```diff
@@ -5,6 +5,7 @@ inherit packagegroup
 PACKAGES = "${PN}"
 
 SDK_TOOLCHAIN_DEPENDS = " \
+  nativesdk-avocado-hitl \
   nativesdk-bmaptool \
   nativesdk-fwup \
   nativesdk-ganesha \
```

Comment:
Not a PR blocker.

Feels like we might be able to delete nativesdk-ganesha from this list given we added nativesdk-avocado-hitl to this list and it brings in ganesha?